IO/IniFileParser.cpp

```diff
@@ -365,9 +366,120 @@ bool IniFileParser::ParseHeader(TiXmlNode * xHeader)
+            if(const char * attribute = node->Attribute("intermediate_goal"); attribute) {
+                std::string in = xmltoa(attribute, "false");
+                std::transform(in.begin(), in.end(), in.begin(), ::tolower);
+
+                if(in == "true") {
+                    _config->AddOptionalOutputOption(OptionalOutput::intermediate_goal);
+                    Log->Write("INFO: intermediate_goal added to output");
+                } else {
+                    Log->Write("INFO: intermediate_goal not added to output");
+                }
+            }
+
+            //check if desired_direction is wanted
+            if(const char * attribute = node->Attribute("desired_direction"); attribute) {
+                std::string in = xmltoa(attribute, "false");
+                std::transform(in.begin(), in.end(), in.begin(), ::tolower);
+
+                if(in == "true") {
+                    _config->AddOptionalOutputOption(OptionalOutput::desired_direction);
+                    Log->Write("INFO: desired_direction added to output");
+                } else {
+                    Log->Write("INFO: desired_direction not added to output");
+                }
+            }
+
+            //check if spotlight is wanted
+            if(const char * attribute = node->Attribute("spotlight"); attribute) {
+                std::string in = xmltoa(attribute, "false");
+                std::transform(in.begin(), in.end(), in.begin(), ::tolower);
+
+                if(in == "true") {
+                    _config->AddOptionalOutputOption(OptionalOutput::spotlight);
+                    Log->Write("INFO: spotlight added to output");
+                } else {
+                    Log->Write("INFO: spotlight not added to output");
+                }
+            }
+
+            //check if router is wanted
+            if(const char * attribute = node->Attribute("router"); attribute) {
+                std::string in = xmltoa(attribute, "false");
+                std::transform(in.begin(), in.end(), in.begin(), ::tolower);
+
+                if(in == "true") {
+                    _config->AddOptionalOutputOption(OptionalOutput::router);
+                    Log->Write("INFO: router added to output");
+                } else {
+                    Log->Write("INFO: router not added to output");
+                }
+            }
+
+            //check if router is wanted
+            if(const char * attribute = node->Attribute("group"); attribute) {
+                std::string in = xmltoa(attribute, "false");
+                std::transform(in.begin(), in.end(), in.begin(), ::tolower);
+
+                if(in == "true") {
+                    _config->AddOptionalOutputOption(OptionalOutput::group);
+                    Log->Write("INFO: group added to output");
+                } else {
+                    Log->Write("INFO: group not added to output");
+                }
+            }
+        }
```

Comment:
Sorry for still using the old logging, is changed in follow up PR (ready to review soon)